recipes/pytest-json/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pytest-json" %}
+{% set version = "0.4.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "8bf4e1be1691f4416bc12b14785b5ad9e842887b0b2b2d61b37dcb555b208630" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    number: 0
```

Comment:
Normally build `number` comes before `script`. Though it's not strictly required to.